src/main/java/com/specialwarriors/conal/common/auth/jwt/repository/RefreshTokenRepository.java

```diff
@@ -0,0 +1,12 @@
+package com.specialwarriors.conal.common.auth.jwt.repository;
+
+import com.specialwarriors.conal.common.auth.jwt.domain.RefreshToken;
+import java.util.Optional;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface RefreshTokenRepository extends JpaRepository<RefreshToken, Long> {
+
+    Optional<RefreshToken> findRefreshTokenByUserId(Long userId);
```

Comment:
```suggestion
    Optional<RefreshToken> findByUserId(Long userId);
```
`RefreshTokenRepository`의 스코프 안이라 굳이 조회 대상 엔티티 이름을 명시할 필요는 없을 것 같은데 명시하신 이유가 있을까요?

Reply:
메서드명에 주체를 넣어서 확실하게 하고자 하는 것이 목표였습니다.
`UserRepository`에서도 `findById`와 같이 쓰는 걸 봤을 때 민재님이 말씀하신 것처럼 수정하는게 통일성 있는 것 같아 수정하겠습니다.